pkg/network/tracer/tracer.go

```diff
@@ -341,10 +341,8 @@ func (t *Tracer) addProcessInfo(c *network.ConnectionStats) {
 	}
 
 	if len(p.Tags) > 0 {
-		c.Tags = make(map[*intern.Value]struct{}, len(p.Tags))
-		for _, t := range p.Tags {
-			c.Tags[t] = struct{}{}
-		}
+		c.Tags = make([]*intern.Value, len(p.Tags))
+		copy(c.Tags, p.Tags)
```

Comment:
Do you even need to do the copy? `p.Tags` doesn't really change after first allocation. I think you can just do `c.Tags = p.Tags`.

Reply:
I wasn't sure on the lifetime of `p.Tags`. If you are confident, I can make that change.

Reply:
It was also so that if `c.Tags` is modified, it doesn't affect `p.Tags`.

Reply:
That is a good point, I think you should keep the copy then.